src/main/java/com/exasol/dynamodb/attributevalue/AttributeValueWrapper.java

```diff
@@ -0,0 +1,50 @@
+package com.exasol.dynamodb.attributevalue;
+
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+
+/**
+ * Wrapper for {@link com.amazonaws.services.dynamodbv2.model.AttributeValue} that accepts {@link AttributeValueVisitor}
+ */
+public class AttributeValueWrapper {
+    private final AttributeValue attributeValue;
+
+    /**
+     * Constructor
+     *
+     * @param attributeValue DynamoDB value
+     */
+    public AttributeValueWrapper(final AttributeValue attributeValue) {
+        this.attributeValue = attributeValue;
+    }
+
+    /**
+     * Accepts the visitor
+     * 
+     * @param visitor {@link AttributeValueVisitor}
+     */
+    public void accept(final AttributeValueVisitor visitor) {
+        if (this.attributeValue.getNULL() != null && this.attributeValue.getNULL()) {
+            visitor.visitNull();
+        } else if (this.attributeValue.getS() != null) {
+            visitor.visitString(this.attributeValue.getS());
```

Comment:
It is weird that attributeValue has these get names.

Reply:
yes.  I created the wrappter to abstract from it...